backend/climateconnect_api/models/language.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+
+
+class Language(models.Model):
+    name = models.CharField(
+        max_length=512, 
+        help_text="Name of a language", 
+        verbose_name="Name"
+    )
+
+    native_name = models.CharField(
+        max_length=512,
+        help_text="Name of language in their native tounge. e.g: German -> Deutsche",
+        verbose_name="Native name",
+        null=True,
+        blank=True
+    )
+
+    language_code = models.CharField(
+        max_length=8,
+        help_text="Short form or Code for each language. e.g.: German/Deutsche has code DE",
```

Comment:
```suggestion
        help_text="Short form or Code for each language. e.g.: German/Deutsch has code de",
```
I think the codes are usually lowercase in software, but doesn't really matter because we can just transform